index.html

```diff
@@ -1129,6 +1129,37 @@ <h3>
           this risk.
         </p>
       </section>
+      <section>
+        <h3>
+          Fingerprinting and Data Leakage
+        </h3>
+        <h4>
+          Browser fingerprinting
+        </h4>
+        <p class="issue" data-number="219">
+          The Digital Credentials API exposes information about which
+          credential exchange protocols are supported by the user agent, which
+          has the potential to be used for browser fingerprinting (see
+          [[[fingerprinting-guidance]]]). We need to add privacy considerations
+          for the involved mitigations here.
+        </p>
+        <h4>
+          Avoiding leaks of credential availability
+        </h4>
+        <p>
+          The Digital Credentials API can not make it possible for websites to
+          learn whether a certain credential is available or not without going
+          through a <a href="#user-permission-and-transparency">user permission
+          flow</a>. Revealing the presence of credentials is a risk to user
```

Comment:
```suggestion
          The Digital Credentials API should not enable sites to
          learn whether a credential is available without first going
          through a <a href="#user-permission-and-transparency">user permission
          flow</a>. Revealing the presence of credentials is a risk to user
```

Reply:
Thanks, I agree with Marcos and those seem like the right words now :)